### PR TITLE
Fix: displaying of disease info

### PR DESCRIPTION
**Pull request by suneettipirneni**

### CODE REVIEW

**Comment by APherwani** [comment]
Screen Shot 2022-04-18 at 10 39 13 PM

Could you add a space between diseases with names that come with commas in the middle? Or better yet, if you're joining two strings for the name just make sure you add ", " as a separator like it was in the db.

**Comment by APherwani** [comment]
For reference, I used SAMD11 for that

File: src/components/TabsCard.tsx [hunk 1]
```diff
@@ -111,63 +111,116 @@ const TabsCard = ({
...
+//           {two}
+//           <br />
+//           {three}
+//           <br /> */}
+//           <a href={two}>{one}</a>
+//         </div>
+//       </>
+//     );
+//   });
+// }
```
Comment (APherwani) [inline src/components/TabsCard.tsx:171]: I think it's fine if you take this giant section of comment out. It was not working anyways.

File: src/components/TabsCard.tsx [hunk 2]
```diff
@@ -111,63 +111,116 @@ const TabsCard = ({
...
-    } else if (diseaseInfo[i] === ']') {
+    if (diseaseInfo[i] === '[') continue;
+    else if (diseaseInfo[i] === ']') {
       list.push(temp);
-      console.log('pushing ' + temp);
-      temp = '';
-      continue;
-    } else if (diseaseInfo[i] === ',') {
-      if (temp !== '') list.push(temp);
+      console.log('pushing ' + temp.trim());
```
Comment (APherwani) [inline src/components/TabsCard.tsx:181]: My bad this was me. I wanted to see if it worked at the time. Could you take it out now that we have it working?

File: src/components/TabsCard.tsx [hunk 3]
```diff
@@ -111,63 +111,116 @@ const TabsCard = ({
...
-    if (i % 3 !== 0) tempList.push(arr[i]);
-    else {
-      res.push(tempList);
-      tempList = [];
+  const diseaseList = [];
+  for (let i = 0; i < list.length; i++) {
+    const subList = list[i].trim().split(', ');
+    for (let j = 0; j < subList.length; j++) {
+      if (subList[j] === '') {
+        console.log('got here');
```
Comment (APherwani) [inline src/components/TabsCard.tsx:193]: take out?

File: src/components/TabsCard.tsx [hunk 4]
```diff
@@ -111,63 +111,116 @@ const TabsCard = ({
...
+
+  return (
+    <div>
+      {diseaseList.map(([id, ...rest]) => {
+        return (
+          <div key={id}>
+            <p>
+              {id}
+              <br />
+              {rest.slice(0, rest.length - 1)}
```
Comment (APherwani) [inline src/components/TabsCard.tsx:214]: add separator between individual words.
might have to write a function for this

**Review by APherwani — APPROVED** [review]
It looks good! If you have time please resolve the comments I left. They're all mostly minor but the separator one would be a good QOL improvement for the user.

Thanks for the hard work!

**Review by Dylans123 — APPROVED** [review]
💯

**Review by joeygiordano13 — COMMENTED** [review]
LGTM

**Pull request merged**